requirements.txt

```diff
@@ -68,6 +68,7 @@ pytz==2018.4
 python-rapidjson==0.8.0
 redis==2.10.6
 redis-py-cluster==1.3.5
+semaphore>=0.4.64,<0.5.0
```

Comment:
I have been very hesitant about this. Finally I think it is not an architectural issue since the dependency is only used on a specific dataset and not in Snuba core. I see it like a dataset specific processor importing its own library it needs to do its job.

Reply:
I'm on the same page here, I think this is fine as long as we isolate these sort of dependencies to the schema and processing layer. If this starts leaking into things like util or the core dataset abstractions this would start to be problematic.